### ISSUE TITLE
QA team test documents discussion

### ISSUE THREAD

**Issue by silySuper** [Magickbase/neuron-public-issues#150]
We can develop some process and documentation specifications to make our work more uniform and standard.I will make some documents firstly，then we can discuss or adjust them according to actual projects by  meeting or issues.
Welcome to the discussion～
@Keith-CY @FrederLu @littleLip520

**Comment by silySuper** [comment]
[测试流程-初版.docx](https://github.com/Magickbase/neuron-public-issues/files/11475080/-.docx)

For the convenience of discussion ，draft will be writen in Chinese ，after we confirm the content，I will change  them
to English.

**Comment by Keith-CY** [comment]
> [测试流程-初版.docx](https://github.com/Magickbase/neuron-public-issues/files/11475080/-.docx)
> 
> For the convenience of discussion ，draft will be writen in Chinese ，after we confirm the content，I will change them to English.

This document has been shared on Google drive(https://docs.google.com/document/d/1Ycm0xllBU1I8OXBVX_O2OUOry9-uW0od)

**Comment by Keith-CY** [comment]
> 1. Google document address of test report template：[docs.google.com/document/d/1bZlH-Tcq6DD0orztP8b4SUhlpxHIaxh9k7Y67mfQsyU/edit](https://docs.google.com/document/d/1bZlH-Tcq6DD0orztP8b4SUhlpxHIaxh9k7Y67mfQsyU/edit)
>    2.Google document address of test case template：[docs.google.com/document/d/1RfP14RmomXicjGP_xtS1Cnyfj_wynWAZhKYq9R0e8Iw/edit](https://docs.google.com/document/d/1RfP14RmomXicjGP_xtS1Cnyfj_wynWAZhKYq9R0e8Iw/edit)

For point 2, `test case template`, I see some fields are covered by platforms mentioned at https://github.com/Magickbase/shaping/issues/48

Maybe we can pick one first and design a report template based on it

**Comment by silySuper** [comment]
YES,most test manange platform can cover test cases fields .OK，QA team will discuss  and decide which one to use  firstly
